app/views/cancelCase/CancelCaseUploadSupportingDocumentationView.scala.html

```diff
@@ -0,0 +1,86 @@
+@*
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+@import config.AppConfig
+@import models.upscan.UpScanInitiateResponse
+
+@this(layout: templates.Layout,
```

Comment:
This page also doesn't seem to have anything specific to Cancel Case on it - no links in the view and all the translations are the same as in the existing `UploadSupportingDocumentationView.scala.html`.